src/Microsoft.IdentityModel.Clients.ActiveDirectory/Core/Exceptions/ExceptionDetail.cs

```diff
@@ -62,7 +64,8 @@ public static ExceptionDetail FromHttpResponse(IHttpWebResponse response)
             {
                 ResponseBody = response?.Body,
                 StatusCode = response != null ? (int)response.StatusCode : -1,
-                HttpResponseHeaders = response?.Headers
+                HttpResponseHeaders = response?.Headers,
```

Comment:
nit: if we record the response here, we don't need to separately record the headers.

Reply:
ignoring for now.

---
In reply to: [267526403](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1530#discussion_r267526403) [](ancestors = 267526403)